src/Image.tsx

```diff
@@ -118,23 +121,27 @@ const ImageInternal: React.FC<ImageProps> = ({
     srcSet,
     useMap,
     alt,
-    className: cn(`${prefixCls}-img`, {
-      [`${prefixCls}-img-placeholder`]: placeholder === true,
-    }),
-    style: height !== undefined ? { height } : undefined,
+    className: cn(
+      `${prefixCls}-img`,
+      {
+        [`${prefixCls}-img-placeholder`]: placeholder === true,
+      },
+      className,
+    ),
+    style: {
+      height,
+      width,
+      ...style
```

Comment:
```suggestion
    style: {
      height,
      ...style
```

Reply:
Keep as it is

Reply:
sorry for the late reply. I have tested the performance in storybook. it should be OK.